test/conformance/testdata/conformance.yaml

```diff
@@ -2180,30 +2180,6 @@
     support running as UID / GID, or privilege escalation.'
   release: v1.15
   file: test/e2e/common/node/security_context.go
-- testname: Sysctls, reject invalid sysctls
-  codename: '[sig-node] Sysctls [LinuxOnly] [NodeFeature:Sysctls] should reject invalid
-    sysctls [MinimumKubeletVersion:1.21] [Conformance]'
-  description: 'Pod is created with one valid and two invalid sysctls. Pod should
-    not apply invalid sysctls. [LinuxOnly]: This test is marked as LinuxOnly since
-    Windows does not support sysctls'
-  release: v1.21
-  file: test/e2e/common/node/sysctl.go
-- testname: Sysctl, test sysctls
-  codename: '[sig-node] Sysctls [LinuxOnly] [NodeFeature:Sysctls] should support sysctls
-    [MinimumKubeletVersion:1.21] [Conformance]'
-  description: 'Pod is created with kernel.shm_rmid_forced sysctl. Kernel.shm_rmid_forced
-    must be set to 1 [LinuxOnly]: This test is marked as LinuxOnly since Windows does
-    not support sysctls'
-  release: v1.21
-  file: test/e2e/common/node/sysctl.go
-- testname: Sysctl, allow specified unsafe sysctls
-  codename: '[sig-node] Sysctls [LinuxOnly] [NodeFeature:Sysctls] should support unsafe
-    sysctls which are actually allowed [MinimumKubeletVersion:1.21] [Conformance]'
-  description: 'Pod is created with kernel.shm_rmid_forced. Should allow unsafe sysctls
-    that are specified. [LinuxOnly]: This test is marked as LinuxOnly since Windows
-    does not support sysctls'
-  release: v1.21
-  file: test/e2e/common/node/sysctl.go
```

Comment:
In @pacoxu's PR only this test was removed from conformance, should we be removing all three? From https://github.com/kubernetes/kubernetes/pull/99734#discussion_r600864919 there was only a question about this particular test, rather than all of the sysctl e2es.

Reply:
Hmm, I'm not sure about this. I'd rather wait for some input in the conformance side.

Reply:
only one test should not be conformance as @pacoxu did. In this PR I suggest to remove the `[NodeFeature:Sysctls]` from the tests description as the feature is GA and having feature tag in test is confusing